src/main/java/htsjdk/samtools/cram/ref/EnaRefService.java

```diff
@@ -11,12 +11,13 @@
 
 public class EnaRefService {
     private static final Log log = Log.getInstance(EnaRefService.class);
-    private static final int HTTP_OK = 200;
+    private static final int HTTP_OK = HttpURLConnection.HTTP_OK;
     private static final int HTTP_FOUND = 302;
```

Comment:
Please remove HTTP_FOUND which is never used and beside it seems to correspond to MOVED_TEMP; I don't know what the "FOUND" actually means here.